src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -448,13 +452,24 @@ <h2 class="m1">Apple</h2>
             [class]="product.selectedColor == 'red' ? 'show' : 'hide'">
         <amp-img src="/img/red_apple_1_1024x682.jpg"
                 width="1024" height="682"
-                layout="responsive">
+                layout="responsive" on="tap:gallery-lightbox"
+                role="button" tabindex="0">
         </amp-img>
         <amp-img src="/img/red_apple_2_1024x793.jpg"
                 width="1024" height="682"
-                layout="responsive">
+                layout="responsive" on="tap:gallery-lightbox"
+                role="button" tabindex="0">
         </amp-img>
     </amp-carousel>
+    <!-- ## Full screen image -->
+    <!-- The `amp-image-lightbox` component allows the user to expand an image to fill the viewport.
+    By clicking on each image from the product gallery, the user can see the product image by using the full screen.
+    Learn more [here](https://ampbyexample.com/components/amp-image-lightbox/) -->
+    <amp-image-lightbox id="gallery-lightbox" layout="nodisplay">
+      <div class="lightbox" on="tap:gallery-lightbox.close" role="button"
```

Comment:
This is not visible. Can you show a close button in the top corner, but also support closing by clicking anywhere?